force-app/main/default/classes/PMT_RefundService.cls

```diff
@@ -62,7 +112,30 @@ public with sharing class PMT_RefundService {
         return config.isIntegrationEnabled();
     }
 
-    public UTIL_Http.Response processRefund(String transactionId) {
+    public PMT_RefundService withOriginalPaymentIds(Set<Id> originalPaymentIds) {
+        this.originalPaymentIds = originalPaymentIds;
+        return this;
+    }
+
+    public PMT_RefundService withOriginalPayments(List<npe01__OppPayment__c> originalPayments) {
+        this.originalPayments = originalPayments;
+        return this;
+    }
+
+    public PMT_RefundService withRefundRecords(List<npe01__OppPayment__c> refundRecords) {
```

Comment:
This function is currently not being used. It will be helpful when we build the Callable API to accept a list of refund records. RefundService is now built in a way where the user can pass in either the Original Payments or the Refund records.